src/preamble.js

```diff
@@ -621,8 +628,12 @@ function stringToUTF8Array(str, outU8Array, outIdx, maxBytesToWrite) {
     }
   }
   // Null-terminate the pointer to the buffer.
-  outU8Array[outIdx] = 0;
-  return outIdx - startIdx;
+	if (withNullTermination_)
```

Comment:
indentation looks wrong. also, we require curly braces with the if/while/etc. bodies are on another line.